queue_services/entity-filer/src/entity_filer/filing_processors/filing_components/legal_entity_info.py

```diff
@@ -38,7 +38,7 @@ def set_corp_type(legal_entity: LegalEntity, legal_entity_info: Dict) -> Dict:
     try:
         entity_type = legal_entity_info.get("legalType")
         if entity_type:
-            legal_entity.entity_type = entity_type
+            legal_entity._entity_type = entity_type
```

Comment:
this has actually reverted to `entity_type` now I believe.  Might need to pull in the latest LE model.